Kentico.Kontent.Management.Tests/Unit/Data/ContentType/ContentType.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
None of the ContentType.json files contains subpages element, shouldnt we include it?